_reports/2023/brink.md

```diff
@@ -0,0 +1,390 @@
+# Organizational Updates
+
+![](img/brink_logo.png)
+
+We led with, and focused, this annual report on the engineering work
+accomplished by Brink-funded developers throughout the year, as their efforts
+directly furthers our mission of strengthening the Bitcoin software, protocol,
+and network. As a non-profit organization with no revenues, we rely entirely on
+community support and our generous sponsors to achieve our mission.
+
+Thanks to the strong support of the community’s contributions, we achieved
+significant growth across our organizational programs. While the bulk of this
+report has largely focused on our engineering output and achievements, we will
+now provide a summary of the financial aspects of our efforts.
+
+*Note: For clarity, the figures provided are approximations based on the cash
+method for 2023. Audited, accrual-based GAAP financials will be available in our
+public 2023 Form 990 filing.*
+
+## Fundraising
+
+Brink’s sponsors donated approximately $2.4 million toward our mission in 2023.
+These contributions came from over 500 different donors and included
+approximately 2.5 BTC in donations.
+
+Thanks to the early financial contributions from our Founding Sponsors–Wences
+Casares, John Pfeffer, and an anonymous donor–Brink’s operating costs were fully
```

Comment:
```suggestion
Thanks to the early financial contributions from our Founding Sponsors---Wences
Casares, John Pfeffer, and an anonymous donor---Brink’s operating costs were fully
```

Nit: Currently renders in the report as an en-dash; switch to triple dash to render as an emdash.

Reply:
If you make this change, maybe a find-and-replace through the file for other en-to-em dash conversions.